code/api/src/config/database.json

```diff
@@ -0,0 +1,18 @@
+{
+  "development": {
+    "username": "ericlarson",
+    "password": null,
+    "database": "crate",
+    "host": "127.0.0.1",
+    "dialect": "postgres",
+    "seederStorage": "sequelize"
+  },
+  "production": {
+    "username": "postgres",
+    "password": null,
+    "database": "crate",
+    "host": "127.0.0.1",
+    "dialect": "postgres",
+    "seederStorage": "sequelize"
+  }
+}
```

Comment:
Can we add this to the .gitignore file?

Reply:
sure, will do

Reply:
As I wrote on Carlos' PR, trying to add to .gitignore after creating a PR is tricky! It took me about an hour to figure it out googling but I don't know what actually made the file disappear from the PR in the end. Sorry, wish I had more information!